src/history/hs.c

```diff
@@ -1051,10 +1051,18 @@ int
 __wt_hs_cursor_position(WT_SESSION_IMPL *session, WT_CURSOR *cursor, uint32_t btree_id,
   const WT_ITEM *key, wt_timestamp_t timestamp, WT_ITEM *user_srch_key)
 {
+    WT_CURSOR_BTREE *cbt;
     WT_DECL_ITEM(srch_key);
     WT_DECL_RET;
+    WT_ITEM hs_key;
+    wt_timestamp_t hs_start_ts;
+    uint64_t hs_counter;
+    uint32_t hs_btree_id;
```

Comment:
I don't love these names. Specifically hs_btree_id makes me think that it's the btree_id for the history store tree, not the btree_id retrieved from a query on the history store.